src/lib.rs

```diff
@@ -164,6 +182,7 @@ pub fn get_dependencies_from_cargo_lock(
     let metadata = metadata_command.exec()?;
 
     let filter = get_node_name_filter(&metadata, &opt)?;
+    let proc_macro_filter = get_proc_macro_node_names(&metadata, &opt);
```

Comment:
The above `fileter` means it should be passed, but `proc_macro_filter` means is shouldn't be passed.
So it may be bit confusable.

Reply:
Agreed, this naming is not good. I've done a renaming pass. I think `proc_macro_exclusions`  is better. If anybody has a better idea, feel free to comment.